slidechain/export.go

```diff
@@ -25,6 +25,15 @@ import (
 	"github.com/stellar/go/xdr"
 )
 
+type pegOutState int
+
+const (
+	pegOutNotYet pegOutState = iota
```

Comment:
another option would be to delete the row from the peg-out table on success or failure — thoughts?

Reply:
and then we would just have the same `unpegged` flag (and retriable errors would just not set that to be true)